argoverse/utils/make_att_files.py

```diff
@@ -161,13 +163,13 @@ def compute_v_a(traj: np.array) -> Tuple[np.array, np.array]:  # traj:Nx3
         if np.sqrt((x_max - x_min) ** 2 + (y_max - y_min) ** 2) < 10:
```

Comment:
could we add a note about why a B-spline is needed in the function comment? I guess for occluded frames to get a constant tick rate (exactly 100 ms)?

Reply:
It's for general temporal smoothing. Not only for the occluded frames. I will add the comment.